std-bits/table/src/main/java/org/enso/table/data/table/join/hashing/CompoundHashJoin.java

```diff
@@ -0,0 +1,90 @@
+package org.enso.table.data.table.join.hashing;
+
+import java.util.List;
+import org.enso.table.data.index.MultiValueIndex;
+import org.enso.table.data.index.UnorderedMultiValueKey;
+import org.enso.table.data.table.join.JoinKind;
+import org.enso.table.data.table.join.JoinResult;
+import org.enso.table.data.table.join.JoinStrategy;
+import org.enso.table.data.table.join.between.SortJoin;
+import org.enso.table.data.table.join.conditions.Between;
+import org.enso.table.data.table.join.conditions.HashableCondition;
+import org.enso.table.problems.ProblemAggregator;
+import org.graalvm.polyglot.Context;
+
+/**
+ * A strategy that uses a hash-map to perform join on the equality conditions.
+ *
+ * <p>It then delegates to {@code sortJoin} to perform the remaining conditions on the matching
```

Comment:
```suggestion
 * <p>It then delegates to {@code SortJoin} to perform the remaining conditions on the matching
```

nit: shouldn't this be the class name so uppercase?

Reply:
Yes. Fixed :)